### DISCUSSION TITLE
In "Browse Data" is there a way to transpose a table so that the fieldnames are on the left and the records run vertically?

### DISCUSSION THREAD

**Discussion by Stampede** [sqlitebrowser/sqlitebrowser#3592] [Q&A] [answered]
I'm not that good with databases, but this tool has been indispensable in developing some python scripts that manipulate databases.

My question is in the title of this thread...it would make it much easier for me if the field names ran down the left side of the screen so I didn't have to scroll the horizontal scrollbar back and forth so much. Is there a way to get the browser to show data this way?

Thank you!

**Comment by chrisjlocke** [comment, marked as answer]
No, thats not a feature.
It may be do-able, but someone would have to code it.  ;)

**Reply by Stampede** [reply]
OK. Thank you for the quick reply!

**Comment by justinclift** [comment]
Yeah, sometimes there's just no good way of presenting a bunch of column based text data.  Well, hiding columns might be the way to go if there's a bunch you don't need.

Hmmm... now that I think about it you *could* create a "VIEW" with just the columns you need.  Those are presented just like a table, but would only show the columns you want.

Views aren't super hard to set up, so might be a useful thing to look into and try out?

* https://youtu.be/h54kCxmSsmY?si=918G51rw_cOp6K6J&t=96
